### DISCUSSION TITLE
Exporting to JSON

### DISCUSSION THREAD

**Discussion by justintoo** [litedb-org/LiteDB#2051] [Q&A] [answered]
The LiteDB Explorer has an export to JSON functionality, but I'm wondering if there's an API available in LiteDB itself to export to JSON or convert the BSON to JSON? I'm specifically looking for a PowerShell solution.

**Comment by lbnascimento** [comment, marked as answer]
@justintoo LiteDB has a way to export collections to JSON:

```C#
var db = new LiteDatabase(...);
db.Execute("select $ into $file('output.json') from mycollection");
```

You can use `db.GetCollectionNames()` to iterate through every collection if you want to export all of the collections.

Regarding using LiteDB in powershell, I have never done it but [apparently it is possible](https://smsagent.blog/2020/11/23/using-a-litedb-portable-database-with-your-powershell-project/).
